recipesesame/src/main/java/com/recipesesame/functions/Handlers.java

```diff
@@ -118,144 +134,219 @@ public static void exploreRecipes(Database database, Scanner scan) throws IOExce
 
 	}
 
+	public static void addRecipeFromFile(Database database, BufferedOutputStream out, Scanner scan) throws IOException {
```

Comment:
We can call this in main and just have it read files from a directory. could be a subdirectory inside the recipes folder so we can reuse whatever file path is set there and not have to worry about inconsistencies in this case.

Reply:
Yea, thats why you can delete or move the file somewhere else after you read it